### DISCUSSION TITLE
Approach for filtering an MVT layer and passing to sublayers

### DISCUSSION THREAD

**Discussion by DanS-arturo** [visgl/deck.gl#6795] [Q&A]
I am a bit stuck on the best way to do what I need.  I am fetching an MVT layer which has a very large amount of data.  I want to be able to filter it before passing it on to another layer.  I tried using a composite layer for this with the thought that at the top level I could load the mvt data and pass the filtered data as sublayer data.  This failed because the composite layer itself doesn't actually fetch the data.

My ultimate goal is:
1. Fetch data for visible tiles
2. Filter data(filters can change dynamically without a need to refetch data from step 1)
3. Render in heatmap

I saw a discussion for something similar but not sure if was the same exact problem.

(Edit) Realized after I read this that I should ahve stated I have no desire to actually display the unfiltered mvt layer.

**Comment by Pessimistress** [comment]
https://github.com/visgl/deck.gl/discussions/6543

**Comment by mapsgeek** [comment]
@DanS-arturo have you got into any solution?
